src/java.base/share/classes/java/time/chrono/Chronology.java

```diff
@@ -765,11 +765,36 @@ public default long epochSecond(int prolepticYear, int month, int dayOfMonth,
      * @throws DateTimeException if any of the values are out of range
      * @since 9
      */
-    public default long epochSecond(Era era, int yearOfEra, int month, int dayOfMonth,
+    default long epochSecond(Era era, int yearOfEra, int month, int dayOfMonth,
                                     int hour, int minute, int second, ZoneOffset zoneOffset) {
         Objects.requireNonNull(era, "era");
         return epochSecond(prolepticYear(era, yearOfEra), month, dayOfMonth, hour, minute, second, zoneOffset);
     }
+
+    /**
+     * Checks if this chronology supports ISO based fields.
+     * <p>
+     * The chronology that supports ISO based fields has the same basic
+     * structure of days and months as the ISO chronology, with month
+     * lengths generally aligned with those in the ISO January to December
+     * definitions.
+     * For example, the Minguo, ThaiBuddhist and Japanese chronologies.
+     *
+     * @implSpec
+     * The default implementation returns {@code false}.
+     *
+     * @return true if the chronology supports ISO based fields
+     * @see IsoChronology
+     * @see JapaneseChronology
+     * @see MinguoChronology
+     * @see ThaiBuddhistChronology
+     * @see java.time.temporal.IsoFields
+     * @since 19
+     */
+    default boolean supportsIsoFields() {
```

Comment:
I'm not a fan of this name, as it is inconsistent with the rest of JSR310 API, which uses an `is` prefix for booleans. I suggested `isIsoLike` because the key question is whether the chronology is "like" ISO. I would also be OK with `isBasedOnIso`, `isDerivedFromIso`, `isIsoBased` or something similar. Another risk here is limiting the method to refer only to `IsoFields`. While that is the use case here, it isn't the case that the only fields that might be affected are in `IsoFields`. Third parties may have  their own fields that are suitable for use with an ISO-like chronology.

Reply:
OK, I propose `isIsoBased()` for the name, which I initially thought of. If there is no objection, I will modify the spec/impl.

Reply:
Is `IsoBased` is fine with me.  "isISOLike" is too vague.

Reply:
Renamed the new method to `isIsoBased()`. Modified the CSR accordingly.